service/policy/db/attribute_values.go

```diff
@@ -33,23 +33,12 @@ func (c PolicyDBClient) CreateAttributeValue(ctx context.Context, attributeID st
 	}
 
 	// Update FQN
-	fqn := c.upsertAttrFqn(ctx, attrFqnUpsertOptions{valueID: createdID})
-	if fqn != "" {
-		c.logger.Debug("created new attribute value FQN",
-			slog.String("value_id", createdID),
-			slog.String("value", value),
-			slog.String("fqn", fqn),
-		)
+	_, err = c.Queries.UpsertAttributeValueFqn(ctx, createdID)
```

Comment:
The response is ignored at the moment, but we could log the rows returned if necessary?  Format of the returned rows looks like:

![image](https://github.com/user-attachments/assets/4574b458-bfcc-4d03-b60d-7b28c6e8002e)

We could also just return the list of FQNs that were upserted...

Reply:
This is fine for now without further enhancement.